src/srctools/game.py

```diff
@@ -84,12 +89,84 @@ def __init__(self, path: Union[str, Path], encoding: str = 'utf8') -> None:
             path = folder / 'update'
             if path.exists():
                 self.search_paths.insert(0, path)
+        
+        mounts_path = self.path / MOUNTSKV # Process mounts.kv file if exists
+        if mounts_path.is_file():
+            with open(mounts_path, encoding=encoding) as m:
+                mountskv = Keyvalues.parse(m)
+
+            self.strata_mounts.extend(mountskv.find_children("Mounts"))
 
     @property
     def root(self) -> Path:
         """Return the game's root folder."""
         return self.path.parent
 
+    # Strata mount compatibility, both in gameinfo and in mounts.kv
+    def parse_strata_mounts(self) -> Tuple[List[Path], List[Path]]:
+        """
+        Parses the mounts in self.strata_mounts and returns two lists of paths.
+        The first should take priority over gameinfo, the second comes after.
```

Comment:
Opposite, first comes after, second comes before

Reply:
Not sure that matches the implementation, `parsed_mounts_heads` ends up being added with `priority=true`, so it'll be added to the start of the `FileSystemChain` and therefore consulted first?

Reply:
Nevermind, I got confused there for a bit, all good.